config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path')
+
+const icons = [
+  'alert-circle',
+  'barcode',
+  'calendar-check',
+  'check-circle',
+  // 'cite',
+  'close',
+  'cog',
+  // 'core-symbol',
+  // 'dashboard',
+  // 'data',
+  'file-alert', // 'document-alert',
+  'file-check', // 'document-success',
+  'file', // 'document',
```

Comment:
@Joozty be aware this is a breaking change. `core-symbol` should not be an icon. The rest will be removed or renamed according to the original icon names from `@mdi/svg` what you may see in the current file. I am working now on finding an proper replacement for everything.

In the future (maybe this, maybe next PR unless we discuss it) design will not ship any icon. The dependent project should build own sprite using provided tools.

Reply:
I added full mapping.